Src/Delivery/studio/built_ins/classic/ISE_RUNTIME.e

```diff
@@ -0,0 +1,81 @@
+class
+	ISE_RUNTIME
+
+feature -- Feature specific to ISE runtime.
+
+	generator_of_type (a_type_id: INTEGER): STRING
+			-- Name of the generating class of current object
+		external
+			"C inline use %"eif_out.h%""
+		alias
+			"return c_generator_of_type(eif_decoded_type($a_type_id));"
+		end
+
+	check_assert (b: BOOLEAN): BOOLEAN
+		external
+			"C use %"eif_copy.h%""
+		alias
+			"c_check_assert"
+		end
+
+	in_assertion: BOOLEAN
```

Comment:
Why did you kept this in_assertion here, and not moved to eif_built_in.h ?
(just wondering)

Reply:
Just because it has two instructions, and I thought it was easier to read in the ISE_RUNTIME.e file than in the 
eif_built_inh.
```
#define eif_builtin_ISE_RUNTIME_in_assertion{\
        GTCX;\ /* --comment */
        return EIF_TEST(in_assertion!=0);\
    }```

Reply:
ok, good reason. I could live with both solutions.